### PR TITLE
Shipments and order shipments endpoints serializations made consistent

### PR DESCRIPTION
**Pull request by paullla**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]

File: tests/Api/Responses/Expected/shop/get_shipping_methods_response.json [hunk 1]
```diff
@@ -18,27 +18,8 @@
...
-                "en_US": {
-                    "@id": "\/api\/v2\/shop\/shipping-method-translations\/@integer@",
-                    "@type": "ShippingMethodTranslation",
-                    "id": @integer@,
-                    "name": "DHL",
-                    "description": @string@
-                }
-            },
-            "name": "DHL",
-            "price": 1000
```
Comment (lchrusciel) [inline tests/Api/Responses/Expected/shop/get_shipping_methods_response.json:40]: the price should be left, as it was

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShippingMethod.xml [hunk 2]
```diff
@@ -41,6 +41,10 @@
             <group>shop:shipping_method:read</group>
         </attribute>
 
+        <attribute name="price">
+            <group>shop:shipping_method:read</group>
+        </attribute>
+
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShippingMethod.xml:47]: Does it change anything?

**Review by lchrusciel — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShippingMethod.xml (hunk 2 above)
Comment (paullla) [reply src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShippingMethod.xml:47]: @lchrusciel this was left here by accident, I'll remove it. Price is added at normalization decorator

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thanks Paula! 🎉

**Pull request merged**